Extensions/TweenBehavior/tweenruntimebehavior.ts

```diff
@@ -239,6 +240,39 @@ namespace gdjs {
       );
     }
 
+    /**
+     * Add an object variable tween.
+     * @param identifier Unique id to identify the tween
+     * @param variable The object variable to store the tweened value
+     * @param toValue End value
+     * @param easingValue Type of easing
+     * @param durationValue Duration in milliseconds
+     * @param destroyObjectWhenFinished Destroy this object when the tween ends
+     */
+    addVariableTween2(
```

Comment:
same here

Reply:
👍

Reply:
I added a detailed comment on the deprecated function